packages/image_picker/ios/Classes/FLTImagePickerImageUtil.m

```diff
@@ -76,6 +76,16 @@ + (UIImage *)scaledImage:(UIImage *)image
                                               scale:1
                                         orientation:UIImageOrientationUp];
 
+  // Swap width and height when source image orientation is left/right
```

Comment:
The code itself is pretty self explanatory, this comment doesn't seem to be necessary. Instead, since it is not clear that why we are doing this swap, maybe we can add some comments to explain the reason.

Reply:
I added a comment to explain the reason for this code.